.gitlab-ci.yml

```
@@ -193,6 +193,27 @@ cargo-check-subkey:
     - BUILD_DUMMY_WASM_BINARY=1 time cargo check --release
     - sccache -s
 
+test-deterministic-wasm:           &test-linux
```

```suggestion
test-deterministic-wasm:
```

We don't need to use anchors - also this anchor will conflict with the `test-linux-stable` job below.